sdk/openai/openai/review/openai.api.md

```diff
@@ -129,10 +163,34 @@ export interface GetEmbeddingsOptions extends RequestOptions {
 }
 
 // @public
+export type ImageGenerationResponseFormat = string;
+
+// @public
+export interface ImageGenerations {
+    created: number;
+    data: ImageLocation[] | ImagePayload[];
+}
+
+// @public
+export interface ImageLocation {
+    url: string;
+}
+
+// @public
+export interface ImagePayload {
+    base64Data: string;
+}
+
+// @public
+export type ImageSize = string;
```

Comment:
Same as above: why is this alias being created? Are we expecting this type to change in the future?

Reply:
similar enum explanation
```@doc("The desired size of the generated images. Must be one of 256x256, 512x512, or 1024x1024.")
@added(ServiceApiVersions.v2023_06_01_Preview)
enum ImageSize {
  @doc("Image size of 256x256.")
  Size256x256: "256x256",

  @doc("Image size of 512x512.")
  Size512x512: "512x512",

  @doc("Image size of 1024x1024.")
  Size1024x1024: "1024x1024",
}
```